### PR TITLE
Updated dependencies (including pyo3 0.18) and using new workflows

### PR DESCRIPTION
**Pull request by kbarkhqs**

### CODE REVIEW

**Review by mlodi-hqs — CHANGES_REQUESTED** [review]
The `pyproject.toml` hasn't been updated (both package and maturin version numbers). Beside that, it looks good.

File: qoqo_aqt/Cargo.toml [hunk 1]
```diff
@@ -22,25 +22,26 @@ doctest = false
...
 roqoqo-aqt = {version="0.6", path="../roqoqo-aqt"}
-qoqo_calculator = {version="1.1.0"}
-qoqo_calculator_pyo3 = {version="1.1.0", default-features=false}
-qoqo = {version="1.2.0", default-features=false}
-roqoqo = {version="1.2.0"}
+
+qoqo_calculator = {version="1.1.1"}
+qoqo_calculator_pyo3 = {version="1.1.1", default-features=false}
+qoqo = {version="1.2.5", default-features=false}
+roqoqo = {version="1.2.5"}
```
[the remark this answers is not in the document]
Comment (kbarkhqs) [reply qoqo_aqt/Cargo.toml]: Changed to 1.1 and 1.2

File: roqoqo-aqt/Cargo.toml [hunk 2]
```diff
@@ -21,14 +21,14 @@ doctest = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-roqoqo = {version="1.2.0", features=["serialize"]}
-qoqo_calculator = { version="1.1.0"}
+roqoqo = {version="1.2.5", features=["serialize"]}
+qoqo_calculator = { version="1.1.1"}
```
[the remark this answers is not in the document]
Comment (kbarkhqs) [reply roqoqo-aqt/Cargo.toml]: Changed as above

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,9 +4,14 @@ This changelog track changes to the qoqo aqt project starting at version 0.3.0
...
 ## Not released
 
+## 0.6.1
+
+* Updated to qoqo 1.2.5 and qoqo_calculator 1.1.1
+
 ## 0.6.0
 
 * Updated to qoqo 1.2.0 and qoqo_calculator 1.1.0
+* Updated to pyo3 0.18
```
[the remark this answers is not in the document]
Comment (kbarkhqs) [reply CHANGELOG.md]: Fixed

**Review by mlodi-hqs — APPROVED** [review]

**Pull request merged**